### ISSUE TITLE
lrn-markdown-editor tabs styling

### ISSUE THREAD

**Issue by jonmann20** [elmsln/issues#819]
I am unable to get the tab styling for `<lrn-markdown-editor>` to work since v2.7.10.  It seems to load `<a11y-tabs>` correctly, but it is not styled.  Also the demo seems to be out of date as it is using `<paper-tabs>`.

https://lrnwebcomponents.github.io/lrn-markdown-editor/components/lrn-markdown-editor/demo/

**Comment by btopro** [comment]
Do you mean this one? https://www.npmjs.com/package/@lrnwebcomponents/lrn-markdown-editor

The repo you linked to for a demo hasn't been maintained in several years as those were the Polymer 1.x elements.

**Comment by jonmann20** [comment]
Yes, I recently tried the v4.0.0 and the same issue exists for me with the tabs not being styled.  If the demo is no longer maintained there, perhaps it could be removed to avoid confusion as well.

**Comment by btopro** [comment]
archived that other repo. it's whatever a11y-tabs kicks out by default which might very well be highly unstyled. We don't support the paper styled stuff anymore but maybe css variables or `::part` can help achieve the styling your looking for:
- https://github.com/elmsln/lrnwebcomponents/blob/master/elements/a11y-tabs/src/a11y-tabs.js
- https://github.com/elmsln/lrnwebcomponents/blob/master/elements/a11y-tabs/lib/a11y-tab.js
